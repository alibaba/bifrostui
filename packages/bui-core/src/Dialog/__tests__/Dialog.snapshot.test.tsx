import { snapshotTest } from 'testing';

describe('Dialog snapshot', () => {
  snapshotTest('Dialog');
});
