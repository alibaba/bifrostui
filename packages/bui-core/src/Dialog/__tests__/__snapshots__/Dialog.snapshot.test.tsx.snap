// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog snapshot Dialog demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    默认为confirm
  </button>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    confirm
  </button>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    等待confirm完成
  </button>
</div>
`;

exports[`Dialog snapshot Dialog demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    默认为confirm
  </button>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    confirm
  </button>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    等待confirm完成
  </button>
</div>
`;

exports[`Dialog snapshot Dialog demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    confirm
  </button>
  <div
    id="container"
  />
</div>
`;

exports[`Dialog snapshot Dialog demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
    onClick={[Function]}
    type="button"
  >
    异步操作执行成功
  </button>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    异步操作执行失败
  </button>
</div>
`;

exports[`Dialog snapshot Dialog demo snapshot 0 5`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    自定义标题和消息
  </button>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    自定义确认按钮文本
  </button>
</div>
`;

exports[`Dialog snapshot Dialog demo snapshot 0 6`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    prompt
  </button>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    传入InputProps
  </button>
</div>
`;
